pkg/manager/aws.go

```diff
@@ -25,24 +30,135 @@ func NewAWS(a aws.Client, k *k8s.Clients, s metrics.Scraper) *AWS {
 }
 
 func (m *AWS) Start(ctx context.Context, errs chan<- error) {
-	ctx, m.cancel = context.WithCancel(ctx)
 	go m.start(ctx, errs)
 }
 
-func (m *AWS) Stop() {
-	if m.cancel == nil {
-		return
-	}
+const (
+	managerInterval = 30 * time.Second
+	nodesPerLicense = 20
```

Comment:
Hardcoding this could be problematic long term, we discussed that elsewhere. We'll see what plans are in future, but I'm not sure how to make it more flexible.